packages/collector/src/agentConnection.js

```diff
@@ -6,24 +6,17 @@
 'use strict';
 
 const { util, uninstrumentedHttp, uninstrumentedFs: fs } = require('@instana/core');
-const http = uninstrumentedHttp.http;
-
 const pathUtil = require('path');
+const circularReferenceRemover = require('./util/removeCircular');
+const agentOpts = require('./agent/opts');
+const cmdline = require('./cmdline');
 
 /** @typedef {import('@instana/core/src/core').InstanaBaseSpan} InstanaBaseSpan */
 
 /** @type {import('@instana/core/src/core').GenericLogger} */
 let logger;
-logger = require('./logger').getLogger('agentConnection', newLogger => {
-  logger = newLogger;
-});
-
-const circularReferenceRemover = require('./util/removeCircular');
-const agentOpts = require('./agent/opts');
-const pidStore = require('./pidStore');
-const cmdline = require('./cmdline');
-
-const cpuSetFileContent = getCpuSetFileContent();
```

Comment:
NOTE: Moved from root initialization to init fn. We should never initialize components on root.